index.html

```diff
@@ -2772,11 +2772,11 @@ <h3>Concatenation of strings</h3>
 <aside class="example" title="Example of bidi generation guidance">
 	<p>Here is an example of a Note that you can include in your specification when recommending that implementations generate displayable [=natural language=] values:</p>
 	<div class="note" role="note" id="bidi-gen-note-example">
-	   <p class="example_note">If values generated for <em><code>_field_name_</code></em> contain or might contain [= bidirectional text =], [=producers=] should ensure that [=consumers=] of the string will display it correctly without having to process or alter the string. One way to do this is via the insertion of Unicode bidirectional controls or, where applicable, markup. In addition, supplying the correct base direction as metadata allows the [=consumer=] to both isolate the string and set its direction, avoiding spillover effects.  For more information see: <a href="https://www.w3.org/International/questions/qa-bidi-unicode-controls"><cite>How to use Unicode controls for bidi text</cite></a></p>
+	   <p class="example_note">If values generated for <em><code>_field_name_</code></em> contain or might contain [= bidirectional text =], [=producers=] should ensure that [=consumers=] of the string will display it correctly without having to process or alter the string. This can be done using markup or via the insertion of Unicode bidirectional controls. In addition, supplying the correct base direction as metadata allows the [=consumer=] to both isolate the string and set its direction, avoiding [=spillover=] effects.  For more information see: <a href="https://www.w3.org/International/questions/qa-bidi-unicode-controls"><cite>How to use Unicode controls for bidi text</cite></a></p>
 	</div>
 </aside>
 
-<p>See also: [:bidi_inline:]</p>
+<p>See also: &#xa7;[[[#inline_changes]]] and &#xa7;[[[#bidi_inline]]]</p>
```

Comment:
If it's better to add the section sign (§), we should probably file an issue in ReSpec and let ReSpec add it automatically in the future.

Reply:
We don't normally add the section sign, and i don't see any reason to add it.

Why are we using &#xa7; rather than § in the source code, anyway?

Reply:
Just realised that i forgot to add `..` to my question about §.  I edited it. It might be easier to understand what i was asking now.

Reply:
And btw, fwiw i was never actually a fan of the use of § by respec.

Reply:
I assumed that you were referring to my use of the entity, but yeah, that makes the comment clearer. I used an entity because I tend to non-ASCII characters in source code. In any case, removing the silcrows.